associations/association.go

```diff
@@ -153,3 +159,9 @@ func isZero(i interface{}) bool {
 	v := reflect.ValueOf(i)
 	return v.Interface() == reflect.Zero(v.Type()).Interface()
 }
+
+// IsZeroOfUnderlyingType will check if the value of anything is the equal to the Zero value of that type
+// using reflect.
```

Comment:
"using reflect" isn't necessary. It should tell about the what, not about the how.